src/Components/ServiceScene/LogsJsonScene.tsx

```diff
@@ -682,10 +780,14 @@ const getStyles = (theme: GrafanaTheme2) => ({
       z-index: 2;
       padding-left: ${theme.spacing(1)};
       align-items: center;
-      overflow-x: scroll;
+      overflow-x: auto;
       overflow-y: hidden;
     }
 
+    > ul > li > span > span > span {
```

Comment:
I think this is the same selector as `drillUpWrapperStyle` in `/explore-logs/src/services/JSONViz.ts`

Reply:
It certainly is.